lib/teleterm/apiserver/apiserver.go

```diff
@@ -92,3 +110,58 @@ type APIServer struct {
 	// grpc is an instance of grpc server
 	grpcServer *grpc.Server
 }
+
+func getGrpcCredentials(cfg Config) (grpc.ServerOption, error) {
+	uri, err := utils.ParseAddr(cfg.HostAddr)
+
+	if err != nil {
+		return nil, trace.BadParameter("invalid host address: %s", cfg.HostAddr)
+	}
+
+	if uri.Network() != "unix" {
```

Comment:
TBH I would stay with the ` != unix` to have the same check as we have in the client. On the electron side it is less confusing to check if the protocol `!== "unix"` rather than `=== ""` (we don't specify the protocol in the address for TCP, because `grpc-js` can't parse it).

Reply:
Ah okay, makes sense.